.travis.yml

```diff
@@ -90,13 +90,14 @@ jobs:
             - libpci-dev
             - libgd-dev
             - libtiff5-dev
-            - libssh-dev
+            - libssh-gcrypt-dev
```

Comment:
I don't understand why we need only `libssh-gcrypt-dev` here and we ask users to install `libssh-dev` in the wiki. Isn't `libssh-gcrypt-dev` sufficient for every cases?

Reply:
Forget my comment `libssh-gcrypt-dev` is mentioned in the wiki.